### DISCUSSION TITLE
ReferenceError: atOptions is not defined

### DISCUSSION THREAD

**Discussion by thanhtoan1196** [livewire/livewire#3834] [7. Bugs (archived)]
I have many of these errors, how do I fix it?
```
ReferenceError: atOptions is not defined
```
![image](https://user-images.githubusercontent.com/16433547/134195802-0f96a526-285d-4a68-b33b-7f4826a53587.png)

**Comment by joshhanley** [comment]
@thanhtoan1196 please don't keep creating multiple discussions for your issues.

I don't see any of these errors in the browser, so if you are catching them in sentry I'd suggest that your logging is too high.

**Reply by thanhtoan1196** [reply]
@joshhanley 
Each discussion I create has a different issue, so I create multiple to keep track of.
"I'd suggest that your logging is too high", what do you mean?
Although the error may not be seen in the browser, the error can occur at some point and is caught by sentry.

**Reply by joshhanley** [reply]
@thanhtoan1196 by logging too high, I mean, generally you can set what you want logged and not logged. I'd suggest changing your logging so it's not picking up every single error.

If you do want the issues fixed though, you're more than welcome to submit a PR with fixes 🙂 

Personally though I would just look into seeing if you can change what issues sentry catches.

Hope this helps!

**Reply by thanhtoan1196** [reply]
OK, thanks a lot
